.github/workflows/deploy.yml

```diff
@@ -53,8 +54,17 @@ jobs:
         if: matrix.os == 'windows-latest'
         run: pip install -r requirements-windows.txt
 
-      - name: 📦 Packaging
-        run: bash ./package_antares_web.sh
+      - name: Initialize installer environment
+        run: |
+          python -m venv .installer
+          source .installer/bin/activate
```

Comment:
This does not seem to work on windows (see workflow failure):
 - the simpler solution will probably be to add `shell: bash` to this step.
 - otherwise syntax seems to be `<venv>\Scripts\Activate.ps1` for powershell which is the default shell on windows (see [doc](https://docs.python.org/3/library/venv.html))